datafusion/core/src/execution/context/mod.rs

```diff
@@ -16,6 +16,13 @@
 // under the License.
 
 //! [`SessionContext`] contains methods for registering data sources and executing queries
+
+mod avro;
```

Comment:
It would be neat (again as a follow on PR) to make avro support also conditionalized